### ISSUE TITLE
What is loss_idt_{A/B}?

### ISSUE THREAD

**Issue by yajain** [junyanz/pytorch-CycleGAN-and-pix2pix#1241]
Rummaging through the code, in the loss functions in cycle_gan_model.py [here](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/f13aab8148bd5f15b9eb47b690496df8dadbab0c/models/cycle_gan_model.py#L162), I saw loss_idt_{A/B}. Can someone explain what it means and why is it there?

Thanks

@junyanz - amazing paper and code needless to say.
